### ISSUE TITLE
Allow bounding box coordinates to be provided in a single argument

### ISSUE THREAD

**Issue by aoles** [rstudio/leaflet#517]
I was wondering whether it would be possible to extend the interface to functions taking `lng1, lat1, lng2, lat2` arguments to allow an alternative way of providing these coordiantes as a single list/vector. This could be implemented e.g. as an additional argument `bbox` to `fitBounds`, `flyToBounds` and `setMaxBounds`. Such flexibility would streamline the code in applications when the bounding box is already provided as a vector.

Looking forward to hearing your thoughts on this.

Cheers,
Andrzej

**Comment by tyluRp** [comment]
I agree. I've been doing [this](https://stackoverflow.com/questions/51392181/setting-boundaries-of-leaflet-with-sf-geometry-in-r/51392873#51392873) but I wish I didn't have to.
